workflows/transport/pika_transport.py

```diff
@@ -194,10 +218,10 @@ def connect(self):
             port = int(
                 self.config.get("--rabbit-port", self.defaults.get("--rabbit-port"))
             )
-
+            vhost = self.config.get("--rabbit-prfx", self.defaults.get("--rabbit-prfx"))
             try:
                 self._conn = pika.BlockingConnection(
-                    pika.ConnectionParameters(host, port, credentials=credentials)
+                    pika.ConnectionParameters(host, port, vhost, credentials)
```

Comment:
Please use keyword arguments rather than relying on argument position - it makes the code more future-proof should `ConnectionParameters` ever change